sw/device/tests/consecutive_irqs_test.c

```diff
@@ -57,7 +43,7 @@ static const buffer_sink_t kPolledUartSink = {
 static void handle_uart_isr(const dif_plic_irq_id_t interrupt_id) {
   const dif_uart_t *uart = &uart0;
 
-  dif_uart_interrupt_t uart_irq;
+  dif_uart_interrupt_t uart_irq = kDifUartInterruptTxWatermark;
```

Comment:
Is there a particular reason you've added defaults to things? If it's to guard against weird uninit behavior, maybe it should be accompanied with a style guide note.